lib/svtplay_dl/utils/getmedia.py

```diff
@@ -136,7 +136,10 @@ def get_one_media(stream):
     except (ValueError, TypeError, KeyError, AttributeError):  # gotta catch em all..
         after_date = None
     try:
-        pub_date = datetime.fromtimestamp(stream.output["publishing_datetime"])
+        if datetime.strptime(stream.output["publishing_datetime"], "%Y-%m-%dT%H:%M:%SZ"):
```

Comment:
this wont work with svtplay.se because the time we get from them we transform their publish time to unixtime as you can see https://github.com/spaam/svtplay-dl/blob/master/lib/svtplay_dl/service/svtplay.py#L343 .
if we send unixtime to this it will throw a `ValueError` and it will only work for tv4play. 

i think it would be better to make tv4play.se also send unixtime.  instead of changing this.